doc/developers.md

```diff
@@ -50,24 +50,16 @@ replay the new events with no more interactions with OpenMRS.
 
 ## Running end-to-end tests
 
-Parquet tools library is used to inspect parquet files. The jar file has been
-included in the project under `/e2e-tests/parquet-tools-1.11.1.jar`
+When a Pull Request is submitted, it triggers a request to Google-owned project running [Cloud Build](https://cloud.google.com/build) to run the build config file defined in [`cloudbuild.yaml`](../cloudbuild.yaml). The CI pipeline can also be run locally, using the Cloud Build local builder. To setup the local builder, follow the instructions [here](https://cloud.google.com/build/docs/build-debug-locally); you will need Docker, and the Google Cloud SDK to install the local builder. Once the local builder is installed, do the following:
```

Comment:
nit: Please limit the line lengths to 100 chars.

Reply:
Please also document where devs should check to see their build progress. Also does the build status communicated back to GitHub and shown to reviewers?

Reply:
Wrote a section in the doc showing how to see progress and read the logs